pypxe/tftp.py

```diff
@@ -18,18 +18,31 @@ class TFTPD:
     '''
     def __init__(self, **serverSettings):
         self.ip = serverSettings.get('ip', '0.0.0.0')
-        self.port = serverSettings.get('port', 69)
+        self.port = serverSettings.get('port', 79)
```

Comment:
Yes, a mistake: should not commited, I have another TFTP running on my machine..